### PR TITLE
Completed html-block-and-inline exercise.

### PR DESCRIPTION
**Pull request by thomas-tahk**
html-block-and-inline

### CODE REVIEW

File: html-block-and-inline/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>HTML Block and Inline</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+</head>
+<body>
+  <h1><b>Blue Americana Snuggie&reg; Blanket</b></h1>
```
Comment (BrettAlbright) [inline html-block-and-inline/index.html]: you don't need the `b` tags in an `h1` tag, the bold text is default styling for all of the `h` elements

File: html-block-and-inline/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <h2><b>by SnuggieStore</b></h2>
```
Comment (BrettAlbright) [inline html-block-and-inline/index.html]: same as above comment

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
looking good, just make sure to remove the `b` tags from the h1 and h2. Once the changes are made, add commit and push again to update the pull request

File: html-block-and-inline/index.html (hunk 1 above)
Comment (thomas-tahk) [reply html-block-and-inline/index.html]: thank you for pointing that out, will reupload changes shortly.

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**